test/models/patient_test.rb

```diff
@@ -78,6 +81,31 @@ class PatientTest < ActiveSupport::TestCase
     end
   end
 
+  describe 'pledge_summary' do
+    it "should return sent and remaining pledge for the next 7 days" do
+      # Doesn't error when there are no pregnancies
+      summary = Patient.pledged_status_summary
+      assert_equal '{:pledged=>0, :sent=>0}', summary.to_s
+      [@patient, @patient2].each do |pt|
+        create :pregnancy, patient: pt, created_by: @user
+      end
+      @patient.update appointment_date: ( Date.today + 4 )
+      @patient.pregnancy.update dcaf_soft_pledge: 300
+      @patient2.update appointment_date: ( Date.today + 8 )
+      @patient2.pregnancy.update dcaf_soft_pledge: 500
+      @patient2.pregnancy.update pledge_sent: true
+      # Returns 0 when there are no scope'd pregnancies
+      summary = Patient.pledged_status_summary(1)
+      assert_equal '{:pledged=>0, :sent=>0}', summary.to_s
```

Comment:
Rather than reassigning the summary variable, what do you think about just straight up comparing `assert_equal (the hash), Patient.pledged_status_summary(1)`?